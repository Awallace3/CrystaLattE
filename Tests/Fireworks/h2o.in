molecule h2o {
O
H 1 1.0
H 1 1.0 2 104.5
}

memory 512 mb
set {
   basis 6-31g
   geom_maxiter 6
}

energy('scf')
