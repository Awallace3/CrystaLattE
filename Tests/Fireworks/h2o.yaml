spec:
  _pass_job_info: true
  _tasks:
  - _fw_name: Psi4Task
  inputfile: 2mer-0+2.in
  ncores: 6
  _launch_dir: /theoryfs2/ds/cborca/Projects/CrystaLattE/Tests/Fireworks/Benzene
