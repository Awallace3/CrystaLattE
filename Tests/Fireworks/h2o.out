
    -----------------------------------------------------------------------
          Psi4: An Open-Source Ab Initio Electronic Structure Package
                               Psi4 1.2a1.dev746 

                         Git: Rev {crystal} c949bea 


    R. M. Parrish, L. A. Burns, D. G. A. Smith, A. C. Simmonett,
    A. E. DePrince III, E. G. Hohenstein, U. Bozkaya, A. Yu. Sokolov,
    R. Di Remigio, R. M. Richard, J. F. Gonthier, A. M. James,
    H. R. McAlexander, A. Kumar, M. Saitow, X. Wang, B. P. Pritchard,
    P. Verma, H. F. Schaefer III, K. Patkowski, R. A. King, E. F. Valeev,
    F. A. Evangelista, J. M. Turney, T. D. Crawford, and C. D. Sherrill,
    J. Chem. Theory Comput. 13(7) pp 3185--3197 (2017).
    (doi: 10.1021/acs.jctc.7b00174)

    -----------------------------------------------------------------------


    Psi4 started on: Sunday, 04 March 2018 08:13PM

    Process ID:  24547
    PSIDATADIR: /theoryfs2/ds/cborca/Gits/psi4/objdir/stage/usr/local/psi4/share/psi4
    Memory:     500.0 MiB
    Threads:    6
    
  ==> Input File <==

--------------------------------------------------------------------------
molecule h2o {
O
H 1 1.0
H 1 1.0 2 104.5
}

memory 512 mb
set {
   basis 6-31g
   geom_maxiter 6
}

energy('scf')
--------------------------------------------------------------------------

  Memory set to 488.281 MiB by Python driver.

*** tstart() called on ds6
*** at Sun Mar  4 20:13:38 2018

   => Loading Basis Set <=

    Name: 6-31G
    Role: ORBITAL
    Keyword: BASIS
    atoms 1   entry O          line   117 file /theoryfs2/ds/cborca/Gits/psi4/objdir/stage/usr/local/psi4/share/psi4/basis/6-31g.gbs 
    atoms 2-3 entry H          line    26 file /theoryfs2/ds/cborca/Gits/psi4/objdir/stage/usr/local/psi4/share/psi4/basis/6-31g.gbs 

    There are an even number of electrons - assuming singlet.
    Specify the multiplicity in the molecule input block.


         ---------------------------------------------------------
                                   SCF
            by Justin Turney, Rob Parrish, Andy Simmonett
                             and Daniel Smith
                              RHF Reference
                        6 Threads,    488 MiB Core
         ---------------------------------------------------------

  ==> Geometry <==

    Molecular point group: c2v
    Full point group: C2v

    Geometry (in Angstrom), charge = 0, multiplicity = 1:

       Center              X                  Y                   Z               Mass       
    ------------   -----------------  -----------------  -----------------  -----------------
           O          0.000000000000     0.000000000000    -0.068516219310    15.994914619560
           H          0.000000000000    -0.790689573744     0.543701060724     1.007825032070
           H          0.000000000000     0.790689573744     0.543701060724     1.007825032070

  Running in c2v symmetry.

  Rotational constants: A =     25.12555  B =     13.37733  C =      8.72955 [cm^-1]
  Rotational constants: A = 753245.06586  B = 401042.16407  C = 261705.25278 [MHz]
  Nuclear repulsion =    8.801465529972067

  Charge       = 0
  Multiplicity = 1
  Electrons    = 10
  Nalpha       = 5
  Nbeta        = 5

  ==> Algorithm <==

  SCF Algorithm Type is DF.
  DIIS enabled.
  MOM disabled.
  Fractional occupation disabled.
  Guess Type is SAD.
  Energy threshold   = 1.00e-06
  Density threshold  = 1.00e-06
  Integral threshold = 0.00e+00

  ==> Primary Basis <==

  Basis Set: 6-31G
    Blend: 6-31G
    Number of shells: 9
    Number of basis function: 13
    Number of Cartesian functions: 13
    Spherical Harmonics?: false
    Max angular momentum: 1

   => Loading Basis Set <=

    Name: (6-31G AUX)
    Role: JKFIT
    Keyword: DF_BASIS_SCF
    atoms 1   entry O          line   221 file /theoryfs2/ds/cborca/Gits/psi4/objdir/stage/usr/local/psi4/share/psi4/basis/cc-pvdz-jkfit.gbs 
    atoms 2-3 entry H          line    51 file /theoryfs2/ds/cborca/Gits/psi4/objdir/stage/usr/local/psi4/share/psi4/basis/cc-pvdz-jkfit.gbs 

  ==> Pre-Iterations <==

   -------------------------------------------------------
    Irrep   Nso     Nmo     Nalpha   Nbeta   Ndocc  Nsocc
   -------------------------------------------------------
     A1         7       7       0       0       0       0
     A2         0       0       0       0       0       0
     B1         2       2       0       0       0       0
     B2         4       4       0       0       0       0
   -------------------------------------------------------
    Total      13      13       5       5       5       0
   -------------------------------------------------------

  ==> Integral Setup <==

  ==> DFJK: Density-Fitted J/K Matrices <==

    J tasked:                  Yes
    K tasked:                  Yes
    wK tasked:                  No
    OpenMP threads:              6
    Integrals threads:           6
    Memory (MB):               366
    Algorithm:                Core
    Integral Cache:           NONE
    Schwarz Cutoff:          1E-12
    Fitting Condition:       1E-12

   => Auxiliary Basis Set <=

  Basis Set: (6-31G AUX)
    Blend: CC-PVDZ-JKFIT
    Number of shells: 42
    Number of basis function: 131
    Number of Cartesian functions: 131
    Spherical Harmonics?: false
    Max angular momentum: 3

  Minimum eigenvalue in the overlap matrix is 7.3923789377E-02.
  Using Symmetric Orthogonalization.

  SCF Guess: Superposition of Atomic Densities via on-the-fly atomic UHF.

  ==> Iterations <==

                           Total Energy        Delta E     RMS |[F,P]|

   @DF-RHF iter   0:   -75.82598117811109   -7.58260e+01   3.47727e-01 
   @DF-RHF iter   1:   -75.95961962893391   -1.33638e-01   4.91554e-02 
   @DF-RHF iter   2:   -75.97524589196847   -1.56263e-02   2.47462e-02 DIIS
   @DF-RHF iter   3:   -75.97997603537418   -4.73014e-03   2.92954e-03 DIIS
   @DF-RHF iter   4:   -75.98012814585525   -1.52110e-04   6.50390e-04 DIIS
   @DF-RHF iter   5:   -75.98014120780391   -1.30619e-05   1.70957e-04 DIIS
   @DF-RHF iter   6:   -75.98014192720699   -7.19403e-07   1.37663e-05 DIIS
   @DF-RHF iter   7:   -75.98014193507390   -7.86692e-09   2.21765e-06 DIIS
   @DF-RHF iter   8:   -75.98014193517751   -1.03611e-10   2.98764e-07 DIIS

  ==> Post-Iterations <==

    Orbital Energies (a.u.)
    -----------------------

    Doubly Occupied:                                                      

       1A1   -20.568888     2A1    -1.334712     1B2    -0.686875  
       3A1    -0.554298     1B1    -0.499284  

    Virtual:                                                              

       4A1     0.193417     2B2     0.287599     3B2     1.041506  
       5A1     1.130379     2B1     1.164432     6A1     1.220999  
       4B2     1.368964     7A1     1.693510  

    Final Occupation by Irrep:
             A1    A2    B1    B2 
    DOCC [     3,    0,    1,    1 ]

  Energy converged.

  @DF-RHF Final Energy:   -75.98014193517751

   => Energetics <=

    Nuclear Repulsion Energy =              8.8014655299720665
    One-Electron Energy =                -122.2750986601641188
    Two-Electron Energy =                  37.4934911950145562
    Total Energy =                        -75.9801419351774996



Properties will be evaluated at   0.000000,   0.000000,   0.000000 Bohr

Properties computed using the SCF density matrix

  Nuclear Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:     1.0191

  Electronic Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:     0.0213

  Dipole Moment: (a.u.)
     X:     0.0000      Y:     0.0000      Z:     1.0404     Total:     1.0404

  Dipole Moment: (Debye)
     X:     0.0000      Y:     0.0000      Z:     2.6444     Total:     2.6444


*** tstop() called on ds6 at Sun Mar  4 20:13:39 2018
Module time:
	user time   =       4.66 seconds =       0.08 minutes
	system time =       0.12 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes
Total time:
	user time   =       4.66 seconds =       0.08 minutes
	system time =       0.12 seconds =       0.00 minutes
	total time  =          1 seconds =       0.02 minutes

*** Psi4 exiting successfully. Buy a developer a beer!
